### PR TITLE
ADD: Update Scripts and Changes to new Setups

### PR DESCRIPTION
**Pull request by NeoPlays**
### Includes:
- Erigon Image Change
- new Relays
- Lido Holesky Allowlist Change

### CODE REVIEW

**Review by daverolo — CHANGES_REQUESTED** [review]
Added a comment to the SSV DKG image change

File: launcher/src/backend/ethereum-services/SSVDKGService.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ outputPath: /data/output`;
     service.setId();
     const workingDir = service.buildWorkingDir(dir);
 
-    const image = "bloxstaking/ssv-dkg";
+    const image = "ssvlabs/ssv-dkg";
```
[the remark this answers is not in the document]
Comment (NeoPlays) [reply launcher/src/backend/ethereum-services/SSVDKGService.js:24]: good point 🙏🏼

[the remark this answers is not in the document]
Comment (daverolo) [reply launcher/src/backend/ethereum-services/SSVDKGService.js]: SSV DKG will get a separate release

**Comment by daverolo** [comment]
This PR will wait as Draft for the SSV node mainnet release, then:
- Update updates.json to the mainnet version
- Do a Stereum release

**Review by gbayasgalan — APPROVED** [review]

**Comment by daverolo** [comment]
JFTR: We decided in MM to remove SSV changes and merge the PR. SSV will get a separate Draft by @NeoPlays when we get an effective date for their mainnet release.

**Pull request merged**